### DISCUSSION TITLE
Specific prices for customer groups

### DISCUSSION THREAD

**Discussion by Max84** [PrestaShop/PrestaShop#35829] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Specific prices are applied only to default group and not to another group.

I think it's a new feature, but I posted it as bug due to the very old report DOC-99 (August 2015) see attachment.
[DOC-99.pdf](https://github.com/PrestaShop/PrestaShop/files/14360894/DOC-99.pdf)

### Expected behavior

Specific prices could be applied if a customer is into a group which is not his default group.

### Steps to reproduce

1. Create a group called "A"
2. Create a customer and set "A" as group. Don't change default group.
3. Create specifc prices to "A" group
4. Display the product on frontoffice logged as this customer, you won't see the "A" group price.

### PrestaShop version(s) where the bug happened

8.1

### PHP version(s) where the bug happened

N/A

**Comment by Max84** [comment]
If the Presta team agree this is a bug or a good new feature, I'll purpose a PR because this feature will be useful for us

**Comment by paulnoelcholot** [comment]
@RosaBenouamer What did you think about that?

**Comment by RosaBenouamer** [comment]
@MatShir :-)
